src/main/resources/config.yml

```diff
@@ -51,6 +51,7 @@ hook:
   betonlangapi: 'true'
   bountifulapi: 'true'
   protocollib: 'true'
+  brewery: 'true'
```

Comment:
You also need to add this to existing configs with `ConfigUpdater` - increase version in _config.yml_ and `version` field of `ConfigUpdater` and then copy one of the `update_from_v00` methods and modify it so it adds the Brewery hook setting.